optimum/intel/openvino/configuration.py

```diff
@@ -646,14 +646,56 @@ def __init__(
             num_samples (`int`, *optional*):
                 The maximum number of samples composing the calibration dataset.
             dataset (`str or List[str]`, *optional*):
-                The dataset used for data-aware optimization with NNCF.
+                The dataset used for data-aware optimization with NNCF. Can be a dataset name (e.g., 'wikitext2')
+                or a string with options (e.g., 'wikitext2:seq_len=128'). The only currently supported option is `seq_len`
```

Comment:
There are some assumptions for seqlen=32 in gptq utils:
https://github.com/huggingface/optimum/blob/main/optimum/gptq/data.py#L128-L129
please check whether this code will be valid for other seq_len.

probably, it makes sense to copy paste this helper code to optimum-intel and extend for other seq_len
```python
    limit = num_samples * seqlen // 4  # ~1k for 128 samples with seqlen=32 to be aligned with optimum
    text = "".join([" \n" if s == "" else s for s in traindata["text"][:limit]])
```

Reply:
Good catch! Added